layouts/_default/list.atom.xml

```diff
@@ -32,14 +32,211 @@
+                        <name>{{ $author.Params.name | plainify }}</name>
+                    {{- else -}}
+                        <name>{{ $name | plainify }}</name>
+                    {{- end -}}
+                    {{- if isset . "uri" }} <!-- explicitly overwrite uri -->
+                        <uri>{{ .uri | safeHTML }}</uri>
+                    {{- else if and $author (isset $author.Params "uri") }} <!-- use different author uri from profile page if found -->
+                        <uri>{{ $author.Params.uri | safeHTML }}</uri>
+                    {{- else if $author }} <!-- link to profile page -->
+                        <uri>{{ $author.Permalink }}</uri>
+                    {{- end -}}
+                    {{- if isset . "email" }} <!-- explicitly overwrite email -->
+                        <email>{{ .email | plainify }}</email>
+                    {{- else if and $author (isset $author.Params "email") }} <!-- use author email from profile page if found -->
+                        <email>{{ $author.Params.email | safeHTML }}</email>
+                    {{- end -}}
+                </author>
+            {{ end }}
+        {{ end }}
+    {{ else if $authors }} <!-- simple author name -->
+        <author><name>{{ $authors | plainify }}</name></author>
+    {{- else }}
+        {{- $author := site.GetPage (printf "/%s/%s" "authors" (.Scratch.Get "superuser_username")) }}
+        {{- if $author.Params.name }} <!-- fallback: a profile page for a superuser was found -->
+            {{ with $author }}
+                <author>
+                    <name>{{ .Params.name | plainify }}</name>
+                    {{ with site.Author.uri }} <!-- author uri may be overwritten for site level -->
+                        <uri>{{ . | safeHTML }}</uri>
+                    {{ else -}}
+                        <uri>{{ .Permalink }}</uri>
+                    {{ end -}}
+                    {{ with site.Author.email }} <!-- author email may be overwritten for site level -->
+                        <email>{{ . | plainify }}</email>
+                    {{ else -}}
+                        {{- with .Params.email }} <!-- default: author email from profile page disabled for privacy reasons -->
+                            <email>{{ . | plainify }}</email>
+                        {{ end -}}
+                    {{ end -}}
+                </author>
+            {{- end -}}
+        {{- end -}}
+    {{- end -}}
+    {{ $contributors := "" }}
+    {{ if isset site "Contributor" }}
+        {{ $contributors = site.Contributor }}
+    {{ else if isset site "Contributors" }}
+        {{ $contributors = site.Contributors }}
+    {{ else if isset site "contributor" }}
+        {{ $contributors = site.contributor }}
+    {{ else if isset site "contributors" }}
+        {{ $contributors = site.contributors }}
+    {{ else if isset site.Params "Contributor" }}
```

Comment:
Use `(or (site.Param "contributor") (site.Param "contributors"))` as suggested for author/authors to replace all these contributor conditions.